### ISSUE TITLE
Hopper detection causing substantial lag in server

### ISSUE THREAD

**Issue by Radplayz** [Navoei/CustomDiscs#85]
Whenever deploying the custom discs plugin, it causes lag in the server for what I can only assume is some sort of hopper detection. Here is a screenshot of the spark profiler

![firefox_6n3bos9KB9](https://github.com/user-attachments/assets/82e806d1-c39b-4899-8bb2-c3fd2d99ea08)

And here is the profiler itself
https://spark.lucko.me/jq2MfZze4f

**Comment by Athar42** [comment]
Nah, it's a function which is run on every chunks loaded (so the more players you have (and also spread across the server), the more you are going to see this happen).

The part of the code involved here should only apply to jukeboxes, to check if a custom music disc is playing, else "stop" the jukebox but...
seeing your profiler result, it seems that it also affect other blocks and 90% of this ressource usage is in fact due to chests 😅

May need some tweaks here 😊
